website/src/content/main-concepts/proxy-to-external-api.mdx

```diff
@@ -214,6 +214,32 @@ exports.handler = async function (request, response) {
 };
 ```
 
+### Decoding the token
```

Comment:
I think we should move this into the `Authenticating requests from the external API` section.
For example: https://auth0.com/docs/tokens/id-tokens/id-token-structure